### PR TITLE
Update README to explain how to run the server

### PR DESCRIPTION
**Pull request by kevindigo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,6 +87,34 @@ Then for each game node (typically one per CPU/core):
...
+
+If you implement or make changes to a card, you can use manual mode 
+to add it to a deck from within a game. Use manual mode, and the command:
+```
+/add-card <card name>
+```
+
+The unit tests require cross-env to be installed:
+```
+npm install cross-env
```
Comment (cryogen) [inline README.md]: This step is not needed, it should be 'npm install' on its own

File: README.md [hunk 2]
```diff
@@ -87,6 +87,34 @@ Then for each game node (typically one per CPU/core):
...
+### Running and Testing
+
+The game server should be accessible by browsing to localhost:4000. 
+
+You can register 2 or more users, to play against yourself. 
+They can have fake email addresses. 
+You can login as both users either from 2 different browsers, or by 
+using an incognito window. 
+
+There is no concept of an 'admin' user.
```
Comment (cryogen) [inline README.md]: This isn't true, but it's not necessary for development

File: README.md (hunk 1 above)
Comment (kevindigo) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (kevindigo) [reply README.md]: Reworded.

**Comment by kevindigo** [comment]
I think I have addressed the PR comments. Let me know if anything else needs to be changed.

**Pull request merged**